packages/@ember/-internals/glimmer/tests/integration/application/debug-render-tree-test.ts

```diff
@@ -788,13 +785,7 @@ if (ENV._DEBUG_RENDER_TREE) {
         ]);
       }
 
-      async [`${testUnless(
-        DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved
-      )} template-only components`]() {
-        expectDeprecation(
-          /resolved templates/,
-          DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isEnabled
-        );
+      async [`@test template-only components`]() {
```

Comment:
This is weird, no? What made you put that there in the first place?

Reply:
Probably a couple things:
- I originally was not updating these tests to use setComponenttemplate
- I changed (per Ed's suggestion) the timing of when the deprecation is thrown to only occur if there is a _found_ resolved template, rather than anytime `layoutFor` is called (this ended up simplifying the implementation in resolver.ts)

Reply:
So should this test remain after the deprecation is removed then?